### DISCUSSION TITLE
[ERROR GRT-0119] what is the limit of congestion when processing early global routing

### DISCUSSION THREAD

**Discussion by imcamino** [The-OpenROAD-Project/OpenROAD-flow-scripts#957] [General]
[ERROR GRT-0119] what is the limit of congestion when processing early global routing

how to fix this problem?
![image](https://user-images.githubusercontent.com/122084694/230056819-f505c721-cdbf-4ef3-8365-47c82a204d8a.png)

**Comment by vijayank88** [comment]
as per `run_openroad.tcl` zip all the required inputs/scripts and share as test case to debug the issue

**Comment by maliberty** [comment]
You can open the rpt file as the error message suggested to get a better sense of where the problem is. Beyond that you should provide a test case
